tests/meson.build

```diff
@@ -1,5 +1,11 @@
-t1 = executable('t1', 't1.c', link_with : pbl_webp, dependencies : [gdkpb, webp])
-t2 = executable('t2', 't2.c', link_with : pbl_webp, dependencies : [gdkpb, webp])
+
+
+t1 = executable('t1', 't1.c',
+    link_with : pbl_webp, dependencies : [gdkpb, webp, webpdemux])
+t2 = executable('t2', 't2.c', link_with : pbl_webp, dependencies : [gdkpb, webp, webpdemux])
+t3 = executable('t3', 't3.c', link_with : pbl_webp, dependencies : [gdkpb, webp, webpdemux])
+t4 = executable('t4', 't4.c', link_with : pbl_webp, dependencies : [gdkpb, webp, webpdemux])
```

Comment:
As mentioned before, split the gtk tests into additional files that are only built if GTK is present using ```gtk3 = dependency('gtk-3.0' ....)``` and then wrapping them with an ```if gtk3/endif``` block.